src/commands/createStaticWebApp/AppLocationStep.ts

```diff
@@ -24,6 +24,7 @@ export class AppLocationStep extends AzureWizardPromptStep<IStaticWebAppWizardCo
         return !wizardContext.appLocation;
     }
 
+    // eslint-disable-next-line @typescript-eslint/require-await
```

Comment:
Rather than disabling this rule, can you make this not async anymore?  Or will the ts error still get in the way?

Reply:
We get a typescript error when the function isn't async:

<img width="941" alt="Screen Shot 2021-02-09 at 10 00 49 AM" src="https://user-images.githubusercontent.com/22795803/107406853-c7e57900-6abd-11eb-941b-5c6d8d404ccd.png">